e2e/runner/bitcoin.go

```diff
@@ -54,6 +55,28 @@ func (r *E2ERunner) ListDeployerUTXOs() ([]btcjson.ListUnspentResult, error) {
 	return utxos, nil
 }
 
+// GetTop20UTXOsForTssAddress returns the top 20 UTXOs for the TSS address
```

Comment:
Why is it necessary?
We should add in comments motivation for implementing function. So far we're been just fetching all utxos for balances